src/lib/y2network/presenters/dns_summary.rb

```diff
@@ -54,6 +52,12 @@ def text
 
     private
 
+      def dhcp_hostname?
+        return false unless config.connections.any? { |c| c.bootproto&.dhcp? }
+
+        hostname.dhcp_hostname && hostname.dhcp_hostname != :none
```

Comment:
I would say that the first part of the conditions is not needed (`nil != :none`). Actually, if `hostname.dhcp_hostname` is nil, this method will return `nil` instead of a boolean.